### DISCUSSION TITLE
Pot | 常见问题

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#990] [General]
# Pot | 常见问题

Pot是一款强大的跨平台划词翻译和OCR软件，支持DeepL、OpenAI、谷歌翻译等多个翻译引擎，具备划词翻译、截图翻译、文字识别、语音合成朗读等功能。简洁的界面设计，快捷的操作方式，是您的最后一款翻译软件。支持离线翻译，确保隐私安全，让跨语言交流更加便捷。

https://pot-app.com/docs/faq.html

**Comment by MidoriiHitsuji** [comment]
使用截图翻译或者文字识别时 屏幕分辨率错误拉伸

**Comment by elarbor** [comment]
点击【翻译回原文】，就回不来了😂
![动画](https://github.com/user-attachments/assets/3a3fb0a4-2543-4817-b8e2-78b9a92226ad)

正常因该点一下成汉字，再点一下就是英语吧😁

**Comment by NikolaSesshoumaru** [comment]
为什么显示“无法注册热键”呢？
